metadata_proxy.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"regexp"
+	"strings"
+)
+
+var (
+	concealedEndpoints = []string{
+		"/0.1/meta-data/attributes/kube-env",
+		"/computeMetadata/v1beta1/instance/attributes/kube-env",
+		"/computeMetadata/v1/instance/attributes/kube-env",
+	}
+	concealedPatterns = []*regexp.Regexp{
+		regexp.MustCompile("/0.1/meta-data/service-accounts/.+/identity"),
+		regexp.MustCompile("/computeMetadata/v1beta1/instance/service-accounts/.+/identity"),
```

Comment:
Should we block out other versions (e.g. "v2alpha1") with these regexps?

Reply:
Addressed in comments below.